src/main/scala/org/broadinstitute/hail/driver/LinearRegressionCommand.scala

```diff
@@ -34,6 +40,12 @@ object LinearRegressionCommand extends Command {
   def run(state: State, options: Options): State = {
     val vds = state.vds
 
+    if (options.minAC < 1)
+      fatal(s"Minumum alternate allele count must be a positive integer, got ${ options.minAC }")
+
+    if (options.minAF < 0d || options.minAF > 1d)
+      fatal(s"Minumum alternate allele frequency must be between 0.0 and 1.0, got ${ options.minAF }")
```

Comment:
Let's use a nullable type for `options.minAF` so we can provide helpful error messages to the user.

```scala
val minAF = Option(options.minAF).map(_.doubleValue())
minAF.foreach { minAF => if (minAF <= 0d || minAF > 1d)
  fatal(s"Minumum alternate allele frequency must lie in (0.0, 1.0], but was ${ minAF }")
}
```

and we can calculate `minAC` something like:

```scala
val minAC = minAF.map(math.max(1, (math.ceil(2 * y.size * options.minAF) + 0.5).toInt))
  .getOrElse(minAC)
```

And actually we probably should make both of them nullable types and enforce that only one of the two is specified.